package/boot/uboot-rockchip/Makefile

```
@@ -222,6 +222,13 @@ define U-Boot/Default/rk3588
   TPL:=rk3588_ddr_lp4_2112MHz_lp5_2400MHz_v1.16.bin
 endef
 
+define U-Boot/nanopc-t6-rk3588
+  $(U-Boot/Default/rk3588)
+  NAME:=NanoPc T6
```

```suggestion
  NAME:=NanoPC T6
```

from the [documentaion](https://wiki.friendlyelec.com/wiki/index.php/NanoPC-T6)